src/manifest.c

```diff
@@ -335,7 +339,8 @@ write_manifest(gzFile f, const struct manifest *mf)
 	for (uint32_t i = 0; i < mf->n_file_infos; i++) {
 		WRITE_INT(4, mf->file_infos[i].index);
 		WRITE_BYTES(mf->hash_size, mf->file_infos[i].hash);
-		WRITE_INT(4, mf->file_infos[i].size);
+		WRITE_INT(8, mf->file_infos[i].size);
+		WRITE_INT(4, mf->file_infos[i].hashed_content_size);
```

Comment:
The description of the disk format at the top of manifest.c needs to be updated with the new fields.